features/admin.groups.v1/components/edit-group/edit-group-users.tsx

```diff
@@ -484,15 +484,19 @@ export const GroupUsersList: FunctionComponent<GroupUsersListProps> = (props: Gr
     return (
         <EmphasizedSegment padded="very" className="list-group-roles-section">
             <Box display="flex" justifyContent="space-between">
-                <div>
-                    <Heading as="h4">
-                        { t("groups:edit.users.heading") }
-                    </Heading>
-                    <Heading subHeading ellipsis as="h6">
-                        { t("groups:edit.users.subHeading") }
-                    </Heading>
-                </div>
-                { !isReadOnly && groupUserList?.totalResults > 0 && (
+                {
+                    (!!searchQuery || groupUserList?.totalResults > 0) &&  (
```

Comment:
Is it intentional to hide the component heading and subheading when there are no groups to be assigned?

Reply:
Yes, because when there are no groups, it should only display the placeholder with the "Assign Group" button